docker-compose.yml

```diff
@@ -52,7 +59,14 @@ services:
       - ./php/conf.d/custom.ini:/usr/local/etc/php/conf.d/custom.ini:ro
       # SSH keys
       - ~/.ssh:/root/.ssh:ro
+    depends_on:
+      - mysql
+      - redis
+      - elasticsearch
     tty: true
+    links:
```

Comment:
These instructions are not needed anymore.

Reply:
I know but, we have problems of communication between these containers and these lines to repair the problem